index.html

```diff
@@ -4201,6 +4207,458 @@ <h3>Device Theft and Impersonation</h3>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf5">PF5.</a> The ability to annotate strings with their language.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf6">PF6.</a> A way to associate datatypes with values such as
+dates and times.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf7">PF7.</a> A facility to express one or more directed graphs,
+such as a social network, in a single document.
+          </td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf8">PF8.</a> Supports signature sets.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf9">PF9.</a> Embeddable in HTML such that search crawlers will
+index the machine-readable content.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+        <tr>
+          <td>
+<a href="#pf10">PF10.</a> Data on the wire is easy to debug and serialize to
+database systems.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf11">PF11.</a> Nesting signed data does not cause data size to
+double for every embedding.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf12">PF12.</a> Proof format supports Zero-Knowledge Proofs.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf13">PF13.</a> Proof format supports arbitrary proofs such as Proof
+of Work, Timestamp Proofs, and Proof of Stake.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf14">PF14.</a> Proofs can be expressed unmodified in other data
+syntaxes such as YAML, N-Quads, and CBOR.
+          </td>
+          <td class="missing">✖</td>
+          <td class="missing">✖</td>
+          <td class="supported">✓</td>
+        </tr>
+
+        <tr>
+          <td>
+<a href="#pf15">PF15.</a> Changing key-value ordering, or introducing
```

Comment:
> There are no use cases where this would be a problem where JWT is used.

Here is one:

You get a JWT in and want to store the JSON in a JSON database. You first check the signature and then, in order to take advantage of the indexing properties in the JSON database, you base-64 decode the data and store it in the database. You are forced to store the JWT header and signature separately.

Time passes, you operate on the JSON data in the database, and then one day, you need to ship the whole header+payload+signature back out to the wire (like when a verifier requests a Verifiable Credential). You pull the JSON object out of the database, base-64 encode it, and send it off. The signature fails because your database rearranges some property-value pair, or injects whitespace in a different way than the entity that generated the signature in the first place.

This is a problem because it means you have to store all of the data twice. Once in base-64 encoded form to preserve the exact serialization that was signed, and once in JSON format, so the database can index the data properly.

Reply:
I dont see that this is a big problem at all. VCs are small in size, so storing the signed VC "as is" makes perfect sense to me. If X.509/ISO presentation layer had recommended this then the distinguished encoding rules would not have been needed.

Reply:
p.s. this is just another example of the CPU vs storage trade off. You prefer using CPU cycles to rebuild the signed VC, others might prefer to store the signed VC.

Reply:
> VCs are small in size, so storing the signed VC "as is" makes perfect sense to me.

Some are small, some (like educational transcripts and healthcare records) are 100KB+ in size. 

> this is just another example of the CPU vs storage trade off

Note that the example I provided (database indexing) made it impossible to index the base-64 encoded information. So, it wasn't an either or trade-off, you were forced to do both things w/ JWTs (no choice) where you only need to do one thing w/ LD-Proofs (choice).